### PR TITLE
 lisav2: remove hardcoded root user for functional test cases

### PR DESCRIPTION
**Pull request by asofron**
Total test cases affected: 21
Tests that fail because of external causes:
- Timesync-VDSO - VDSO not supported by kernel
- KVP-MaxRead - Fails  get correct version if not RHEL distro

Libraries/CommonFunctions.psm1 - 2 TC (BVT-CORE-RELOAD-MODULES-SMP, STRESSTEST-RELOAD-MODULES-UP)
Testscripts/Windows/BVT-VERIFY-KVP-DAEMON-STATUS.ps1 -1 TC BVT-VERIFY-KVP-DAEMON-STATUS
Testscripts/Windows/DM-HOTADD-STRESS-APP.ps1 - 1 TC DYNAMIC-MEMORY-HOTADD-STRESS-APP
Testscripts/Windows/FCOPY-DISABLE-ENABLE.ps1 - 1 TC FCOPY-DISABLE-ENABLE
Testscripts/Windows/FCOPY-LARGE-FILE.ps1 -1 TC FCOPY-DISABLE-ENABLE
Testscripts/Windows/FCOPY-NEGATIVE.ps1 -1 TC FCOPY-NEGATIVE
Testscripts/Windows/FCOPY-REPEATED-DELETE.ps1 - 1TC FCOPY-REPEATED-DELETE
Testscripts/Windows/Heartbeat-PausedCritical.ps1 - 1TC PAUSED-CRITICAL-HEARTBEAT
Testscripts/Windows/KDUMP-TestScript.ps1 6TC - KDUMP-CRASH-SINGLE-CORE , KDUMP-CRASH-SMP, KDUMP-CRASH-NMI,KDUMP-CRASH-AUTO-SIZE, KDUMP-CRASH-AUTO-SIZE ,KDUMP-CRASH-16-CORES
Testscripts/Windows/KVP-MaxRead.ps1 1 TC - KVP-MAX-READ
Testscripts/Windows/KVP-PushKeyValues.ps1 -1TC KVP-PUSH-KEY-VALUES
Testscripts/Windows/Max-vCPU.ps1 - 1 TC MAX-VCPU
Testscripts/Windows/SQM-Basic.ps1 - 1 TC SQM-BASIC
Testscripts/Windows/STORAGE-HotUnplug.ps1 - 1TC STORAGE-VHDX-HOT-PLUG-UNPLUG-SCSI
Testscripts/Windows/STRESS-MTU-Netvsc-Reload.ps1 -1TC STRESSTEST-CHANGE-MTU-RELOAD-NETVSC
Testscripts/Windows/Timesync-VDSO.ps1 -1TC TIMESYNC-VDSO

### CODE REVIEW

**Comment by asofron** [comment]
**Hyper-V/ Ubuntu :**

[hyperv] [LISAv2 Test Results Summary]
[hyperv] Test Run On           : 12/13/2018 20:35:16
[hyperv] VHD Under Test        : ubuntu_18.04.1.vhdx
[hyperv] Total Test Cases      : 4 (4 Pass, 0 Fail, 0 Abort)
[hyperv] Total Time (dd:hh:mm) : 0:0:28
[hyperv] XML File              : TestConfiguration
[hyperv] 
[hyperv]    ID TestCaseName                                                 TestResult TestDuration(in minutes) 
[hyperv] ------------------------------------------------------------------------------------------------------
[hyperv]     1 FCOPY-LARGE-FILE                                                   PASS                11.15 
[hyperv]     2 FCOPY-REPEATED-DELETE                                              PASS                 4.93 
[hyperv]     3 FCOPY-DISABLE-ENABLE                                               PASS                 9.38 
[hyperv]     4 FCOPY-NEGATIVE                                                     PASS                 3.38 

[hyperv] [LISAv2 Test Results Summary]
[hyperv] Test Run On           : 12/13/2018 15:00:59
[hyperv] VHD Under Test        : ubuntu_18.04.1.vhdx
[hyperv] Total Test Cases      : 6 (6 Pass, 0 Fail, 0 Abort)
[hyperv] Total Time (dd:hh:mm) : 0:0:45
[hyperv] XML File              : TestConfiguration
[hyperv] 
[hyperv]    ID TestCaseName                                                 TestResult TestDuration(in minutes) 
[hyperv] ------------------------------------------------------------------------------------------------------
[hyperv]     1 KDUMP-CRASH-SINGLE-CORE                                            PASS                 8.82 
[hyperv]     2 KDUMP-CRASH-SMP                                                    PASS                 7.02 
[hyperv]     3 KDUMP-CRASH-NMI                                                    PASS                 7.87 
[hyperv]     4 KDUMP-CRASH-AUTO-SIZE                                              PASS                 6.66 
[hyperv]     5 KDUMP-CRASH-DIFFERENT-VCPU                                         PASS                 6.61 
[hyperv]     6 KDUMP-CRASH-16-CORES                                               PASS                 8.05 


[hyperv] [LISAv2 Test Results Summary]
[hyperv] Test Run On           : 12/13/2018 15:23:03
[hyperv] VHD Under Test        : ubuntu_18.04.1.vhdx
[hyperv] Total Test Cases      : 5 (4 Pass, 0 Fail, 1 Abort)
[hyperv] Total Time (dd:hh:mm) : 0:0:47
[hyperv] XML File              : TestConfiguration
[hyperv] 
[hyperv]    ID TestCaseName                                                 TestResult TestDuration(in minutes) 
[hyperv] ------------------------------------------------------------------------------------------------------
[hyperv]     1 SQM-BASIC                                                          PASS                 2.72 
[hyperv]     2 STORAGE-VHDX-HOT-PLUG-UNPLUG-SCSI                                  PASS                 4.89 
[hyperv]     3 MAX-VCPU                                                           PASS                 6.72 
[hyperv]     4 TIMESYNC-VDSO                                                   Aborted                 2.52 
[hyperv]     5 STRESSTEST-CHANGE-MTU-RELOAD-NETVSC                                PASS                30.79 
[hyperv] TIMESYNC-VDSO :[INFO ] Info: Current VM Linux kernel version does not support vDSO feature.

[hyperv] [LISAv2 Test Results Summary]
[hyperv] Test Run On           : 12/13/2018 15:47:11
[hyperv] VHD Under Test        : ubuntu_18.04.1.vhdx
[hyperv] Total Test Cases      : 2 (2 Pass, 0 Fail, 0 Abort)
[hyperv] Total Time (dd:hh:mm) : 0:0:26
[hyperv] XML File              : TestConfiguration
[hyperv] 
[hyperv]    ID TestCaseName                                                 TestResult TestDuration(in minutes) 
[hyperv] ------------------------------------------------------------------------------------------------------
[hyperv]     1 DYNAMIC-MEMORY-HOTADD-STRESS-APP                                   PASS                10.36 
[hyperv]     2 PAUSED-CRITICAL-HEARTBEAT                                          PASS                15.81 
[hyperv] 


[hyperv] [LISAv2 Test Results Summary]
[hyperv] Test Run On           : 12/13/2018 16:34:21
[hyperv] VHD Under Test        : ubuntu_18.04.1.vhdx
[hyperv] Total Test Cases      : 2 (1 Pass, 0 Fail, 1 Abort)
[hyperv] Total Time (dd:hh:mm) : 0:0:5
[hyperv] XML File              : TestConfiguration
[hyperv] 
[hyperv]    ID TestCaseName                                                 TestResult TestDuration(in minutes) 
[hyperv] ------------------------------------------------------------------------------------------------------
[hyperv]     1 KVP-PUSH-KEY-VALUES                                                PASS                 2.71 
[hyperv]     2 KVP-MAX-READ                                                    Aborted                 2.43 
[hyperv] 
[hyperv] 2 KVP-MAX-READ :
[hyperv] 12/13/2018 16:39:12 : [INFO ] Kernels older than 3.10.0-862 require LIS-4.x drivers.
[hyperv] 12/13/2018 16:39:12 : [ERROR] Error: No LIS-4.x drivers detected. Skipping test.



[hyperv] [LISAv2 Test Results Summary]
[hyperv] Test Run On           : 12/13/2018 16:05:52
[hyperv] VHD Under Test        : ubuntu_18.04.1.vhdx
[hyperv] Total Test Cases      : 1 (1 Pass, 0 Fail, 0 Abort)
[hyperv] Total Time (dd:hh:mm) : 0:0:14
[hyperv] XML File              : TestConfiguration
[hyperv] 
[hyperv]    ID TestCaseName                                                 TestResult TestDuration(in minutes) 
[hyperv] ------------------------------------------------------------------------------------------------------
[hyperv]     1 BVT-CORE-RELOAD-MODULES-SMP                                        PASS                14.97 


[hyperv] [LISAv2 Test Results Summary]
[hyperv] Test Run On           : 12/13/2018 16:24:18
[hyperv] VHD Under Test        : ubuntu_18.04.1.vhdx
[hyperv] Total Test Cases      : 1 (1 Pass, 0 Fail, 0 Abort)
[hyperv] Total Time (dd:hh:mm) : 0:1:45
[hyperv] XML File              : TestConfiguration
[hyperv] 
[hyperv]    ID TestCaseName                                                 TestResult TestDuration(in minutes) 
[hyperv] ------------------------------------------------------------------------------------------------------
[hyperv]     1 STRESSTEST-RELOAD-MODULES-UP                                       PASS               105.42 


**Azure /Ubuntu :**


[azure] [LISAv2 Test Results Summary]
[azure] Test Run On           : 12/13/2018 16:18:20
[azure] ARM Image Under Test  : Canonical : UbuntuServer : 18.04-LTS : latest
[azure] Total Test Cases      : 1 (1 Pass, 0 Fail, 0 Abort)
[azure] Total Time (dd:hh:mm) : 0:0:3
[azure] XML File              : TestConfiguration
[azure] 
[azure]    ID TestCaseName                                                 TestResult TestDuration(in minutes) 
[azure] ------------------------------------------------------------------------------------------------------
[azure]     1 BVT-VERIFY-KVP-DAEMON-STATUS                                       PASS                 3.22 
[azure] 

[azure] [LISAv2 Test Results Summary]
[azure] Test Run On           : 12/13/2018 16:27:28
[azure] ARM Image Under Test  : Canonical : UbuntuServer : 18.04-LTS : latest
[azure] Total Test Cases      : 5 (5 Pass, 0 Fail, 0 Abort)
[azure] Total Time (dd:hh:mm) : 0:0:29
[azure] XML File              : TestConfiguration
[azure] 
[azure]    ID TestCaseName                                                 TestResult TestDuration(in minutes) 
[azure] ------------------------------------------------------------------------------------------------------
[azure]     1 KDUMP-CRASH-SINGLE-CORE                                            PASS                 5.83 
[azure]     2 KDUMP-CRASH-SMP                                                    PASS                 6.09 
[azure]     3 KDUMP-CRASH-AUTO-SIZE                                              PASS                 5.88 
[azure]     4 KDUMP-CRASH-DIFFERENT-VCPU                                         PASS                 5.58 
[azure]     5 KDUMP-CRASH-16-CORES                                               PASS                 5.88 

[azure] [LISAv2 Test Results Summary]
[azure] Test Run On           : 12/13/2018 16:42:37
[azure] ARM Image Under Test  : Canonical : UbuntuServer : 18.04-LTS : latest
[azure] Total Test Cases      : 1 (1 Pass, 0 Fail, 0 Abort)
[azure] Total Time (dd:hh:mm) : 0:0:14
[azure] XML File              : TestConfiguration
[azure] 
[azure]    ID TestCaseName                                                 TestResult TestDuration(in minutes) 
[azure] ------------------------------------------------------------------------------------------------------
[azure]     1 BVT-CORE-RELOAD-MODULES-SMP                                        PASS                14.45 
[azure] 
[azure] 


[azure] [LISAv2 Test Results Summary]
[azure] Test Run On           : 12/13/2018 17:08:31
[azure] ARM Image Under Test  : Canonical : UbuntuServer : 18.04-LTS : latest
[azure] Total Test Cases      : 1 (1 Pass, 0 Fail, 0 Abort)
[azure] Total Time (dd:hh:mm) : 0:1:45
[azure] XML File              : TestConfiguration
[azure] 
[azure]    ID TestCaseName                                                 TestResult TestDuration(in minutes) 
[azure] ------------------------------------------------------------------------------------------------------
[azure]     1 STRESSTEST-RELOAD-MODULES-UP                                       PASS               105.69 
[azure] 
[azure]

File: Libraries/CommonFunctions.psm1 [hunk 1]
```diff
@@ -2739,7 +2739,7 @@ function Check-Result{
 
 	while ($timeout -ne 0 )
 	{
-		Write-Output "yes" | .\tools\pscp.exe  -v -2 -unsafe -pw $vmPassword -q -P ${vmPort} root@${ipv4}:${stateFile} ${localStateFile} #| out-null
+		Write-Output "yes" | .\tools\pscp.exe  -v -2 -unsafe -pw $vmPassword -q -P ${vmPort} $user@${ipv4}:${stateFile} ${localStateFile} #| out-null
```
Comment (LiliDeng) [inline Libraries/CommonFunctions.psm1]: Below is method define, and invoked by this way, only one file invoke this method.
Check-Result -VmIp $Ipv4 -VmPort $VMPort -User $VMUserName -Password $VMPassword

function Check-Result{
	param(
		[String] $vmPassword,
		[String] $vmPort,
		[String] $ipv4
	)

File: Testscripts/Windows/BVT-VERIFY-KVP-DAEMON-STATUS.ps1 [hunk 2]
```diff
@@ -13,7 +13,7 @@ function Main {
         Write-LogInfo "Executing : $($currentTestData.testScriptPs1)"
         Set-Content -Value "**************$($currentTestData.testName)******************" -Path "$logDir\$($currentTestData.testName)_Log.txt"
         Write-LogInfo "Verifcation of KVP Daemon status is started.."
-        $kvpStatus = Run-LinuxCmd -username "root" -password $password -ip $AllVMData.PublicIP -port $AllVMData.SSHPort -command "pgrep -lf 'hypervkvpd|hv_kvp_daemon'"
+        $kvpStatus = Run-LinuxCmd -username  $user -password $password -ip $AllVMData.PublicIP -port $AllVMData.SSHPort -command "pgrep -lf 'hypervkvpd|hv_kvp_daemon'" -RunAsSudo
```
Comment (LiliDeng) [inline Testscripts/Windows/BVT-VERIFY-KVP-DAEMON-STATUS.ps1]: -username $user one extra space between '-username' and $user

File: Testscripts/Windows/DM-HOTADD-STRESS-APP.ps1 [hunk 3]
```diff
@@ -25,7 +25,7 @@ $scriptBlock = {
         $cmdToVM = @"
 #!/bin/bash
         if [ ! -e /proc/meminfo ]; then
-            echo ConsumeMemory: no meminfo found. Make sure /proc is mounted >> /root/HotAdd.log 2>&1
+            echo ConsumeMemory: no meminfo found. Make sure /proc is mounted >>/home/$user/HotAdd.log 2>&1
```
Comment (LiliDeng) [inline Testscripts/Windows/DM-HOTADD-STRESS-APP.ps1]: >> /home/$user

File: Testscripts/Windows/FCOPY-LARGE-FILE.ps1 [hunk 4]
```diff
@@ -114,10 +114,10 @@ if ($createfile -notlike "File *testfile-*.file is created") {
 	return "FAIL"
 }
 # Verifying if /mnt folder on guest exists; if not, it will be created
-.\Tools\plink.exe -C -pw $VMPassword -P $VMPort $VMUserName@$Ipv4 "[ -d /mnt ]"
+Run-LinuxCmd -username $VMUserName -password $VMPassword -ip $Ipv4 -port $VMPort "[ -d /mnt ]" -runAsSudo
```
Comment (LiliDeng) [inline Testscripts/Windows/FCOPY-LARGE-FILE.ps1]: -command "[ -d /mnt ]" ?

File: Testscripts/Windows/FCOPY-REPEATED-DELETE.ps1 [hunk 5]
```diff
@@ -102,7 +102,7 @@ else {
 }
 
 # Verifying if /tmp folder on guest exists; if not, it will be created
-.\Tools\plink.exe -C -pw $VMPassword -P $VMPort $VMUserName@$Ipv4 "[ -d /tmp ]"
+Run-LinuxCmd -username $VMUserName -password $VMPassword -ip $Ipv4 -port $VMPort "[ -d /tmp ]" -runAsSudo
```
Comment (LiliDeng) [inline Testscripts/Windows/FCOPY-REPEATED-DELETE.ps1]: ditto

File: Libraries/CommonFunctions.psm1 (hunk 1 above)
Comment (asofron) [reply Libraries/CommonFunctions.psm1]: Fixed

File: Testscripts/Windows/DM-HOTADD-STRESS-APP.ps1 (hunk 3 above)
Comment (asofron) [reply Testscripts/Windows/DM-HOTADD-STRESS-APP.ps1]: Fixed

File: Testscripts/Windows/STRESS-MTU-Netvsc-Reload.ps1 [hunk 6]
```diff
@@ -59,10 +57,10 @@ function Main {
     }
 
     Add-Content $scriptPath "$RELOAD_COMMAND"
-    Copy-RemoteFiles -uploadTo $Ipv4 -port $VMPort -password $VMPassword -username $rootUser `
+    Copy-RemoteFiles -uploadTo $Ipv4 -port $VMPort -password $VMPassword -username $VMUserName `
         -files $scriptPath -upload
-    $null = Run-LinuxCmd -username $rootUser -password $VMPassword -ip $Ipv4 -port $VMPort `
-        -command "dos2unix reload_netvsc.sh && sleep 5 && bash ~/reload_netvsc.sh" -RunInBackGround
+    $null = Run-LinuxCmd -username $VMUserName -password $VMPassword -ip $Ipv4 -port $VMPort `
+        -command "dos2unix reload_netvsc.sh && sleep 5 && bash ~/reload_netvsc.sh" -RunInBackGround  -RunAsSudo
```
Comment (LiliDeng) [inline Testscripts/Windows/STRESS-MTU-Netvsc-Reload.ps1]: ~/reload_netvsc.sh this file is under root?

File: Testscripts/Windows/FCOPY-LARGE-FILE.ps1 (hunk 4 above)
Comment (asofron) [reply Testscripts/Windows/FCOPY-LARGE-FILE.ps1]: Fixed

File: Testscripts/Windows/FCOPY-LARGE-FILE.ps1 [hunk 7]
```diff
@@ -114,10 +114,10 @@ if ($createfile -notlike "File *testfile-*.file is created") {
...
 if (-not $?){
     Write-LogInfo "Folder /mnt not present on guest. It will be created"
-    .\Tools\plink.exe -C -pw $VMPassword -P $VMPort $VMUserName@$Ipv4 "mkdir /mnt"
+    Run-LinuxCmd -username $VMUserName -password $VMPassword -ip $Ipv4 -port $VMPort "mkdir /mnt" -runAsSudo
```
[the remark this answers is not in the document]
Comment (asofron) [reply Testscripts/Windows/FCOPY-LARGE-FILE.ps1]: Fixed

File: Testscripts/Windows/FCOPY-REPEATED-DELETE.ps1 (hunk 5 above)
Comment (asofron) [reply Testscripts/Windows/FCOPY-REPEATED-DELETE.ps1]: Fixed

File: Testscripts/Windows/BVT-VERIFY-KVP-DAEMON-STATUS.ps1 (hunk 2 above)
Comment (asofron) [reply Testscripts/Windows/BVT-VERIFY-KVP-DAEMON-STATUS.ps1]: fixed

File: Testscripts/Windows/STRESS-MTU-Netvsc-Reload.ps1 (hunk 6 above)
Comment (asofron) [reply Testscripts/Windows/STRESS-MTU-Netvsc-Reload.ps1]: no it`s under user an run for there

**Comment by juhlee-microsoft** [comment]
@LiliDeng please sign off if you don't have any other question by Monday morning.

File: Testscripts/Windows/BVT-VERIFY-KVP-DAEMON-STATUS.ps1 [hunk 8]
```diff
@@ -13,7 +13,7 @@ function Main {
         Write-LogInfo "Executing : $($currentTestData.testScriptPs1)"
         Set-Content -Value "**************$($currentTestData.testName)******************" -Path "$logDir\$($currentTestData.testName)_Log.txt"
         Write-LogInfo "Verifcation of KVP Daemon status is started.."
-        $kvpStatus = Run-LinuxCmd -username "root" -password $password -ip $AllVMData.PublicIP -port $AllVMData.SSHPort -command "pgrep -lf 'hypervkvpd|hv_kvp_daemon'"
+        $kvpStatus = Run-LinuxCmd -username $user -password $password -ip $AllVMData.PublicIP -port $AllVMData.SSHPort -command "pgrep -lf 'hypervkvpd|hv_kvp_daemon'" -RunAsSudo
```
Comment (LiliDeng) [inline Testscripts/Windows/BVT-VERIFY-KVP-DAEMON-STATUS.ps1]: Please rebase the file before merge it

File: Testscripts/Windows/STRESS-MTU-Netvsc-Reload.ps1 (hunk 6 above)
[the remark this answers is not in the document]
Comment (asofron) [reply Testscripts/Windows/STRESS-MTU-Netvsc-Reload.ps1]: removed ~/

[hyperv] [LISAv2 Test Results Summary]
[hyperv] Test Run On           : 12/17/2018 09:19:59
[hyperv] VHD Under Test        : Ubuntu_18.04.1.vhdx
[hyperv] Total Test Cases      : 1 (1 Pass, 0 Fail, 0 Abort)
[hyperv] Total Time (dd:hh:mm) : 0:0:26
[hyperv] XML File              : TestConfiguration
[hyperv] 
[hyperv]    ID TestCaseName                                                 TestResult TestDuration(in minutes) 
[hyperv] ------------------------------------------------------------------------------------------------------
[hyperv]     1 STRESSTEST-CHANGE-MTU-RELOAD-NETVSC                                PASS                26.41

File: Testscripts/Windows/BVT-VERIFY-KVP-DAEMON-STATUS.ps1 (hunk 8 above)
Comment (asofron) [reply Testscripts/Windows/BVT-VERIFY-KVP-DAEMON-STATUS.ps1]: [azure] [LISAv2 Test Results Summary]
[azure] Test Run On           : 12/17/2018 10:38:02
[azure] ARM Image Under Test  : Canonical : UbuntuServer : 18.04-LTS : latest
[azure] Total Test Cases      : 1 (1 Pass, 0 Fail, 0 Abort)
[azure] Total Time (dd:hh:mm) : 0:0:3
[azure] XML File              : TestConfiguration
[azure] 
[azure]    ID TestCaseName                                                 TestResult TestDuration(in minutes) 
[azure] ------------------------------------------------------------------------------------------------------
[azure]     1 BVT-VERIFY-KVP-DAEMON-STATUS                                       PASS                 3.37

File: Testscripts/Windows/FCOPY-REPEATED-DELETE.ps1 (hunk 5 above)
Comment (asofron) [reply Testscripts/Windows/FCOPY-REPEATED-DELETE.ps1]: [hyperv] Test Run On           : 12/17/2018 10:44:23
[hyperv] VHD Under Test        : Ubuntu_18.04.vhdx
[hyperv] Total Test Cases      : 1 (1 Pass, 0 Fail, 0 Abort)
[hyperv] Total Time (dd:hh:mm) : 0:0:11
[hyperv] XML File              : TestConfiguration
[hyperv] 
[hyperv]    ID TestCaseName                                                 TestResult TestDuration(in minutes) 
[hyperv] ------------------------------------------------------------------------------------------------------
[hyperv]     1 FCOPY-REPEATED-DELETE                                              PASS                11.71 
[hyperv] 
[hyperv]

File: Testscripts/Windows/FCOPY-LARGE-FILE.ps1 (hunk 7 above)
[the remark this answers is not in the document]
Comment (asofron) [reply Testscripts/Windows/FCOPY-LARGE-FILE.ps1]: [hyperv] [LISAv2 Test Results Summary]
[hyperv] Test Run On           : 12/17/2018 10:43:22
[hyperv] VHD Under Test        : Ubuntu_18.04.1.vhdx
[hyperv] Total Test Cases      : 1 (1 Pass, 0 Fail, 0 Abort)
[hyperv] Total Time (dd:hh:mm) : 0:0:11
[hyperv] XML File              : TestConfiguration
[hyperv] 
[hyperv]    ID TestCaseName                                                 TestResult TestDuration(in minutes) 
[hyperv] ------------------------------------------------------------------------------------------------------
[hyperv]     1 FCOPY-LARGE-FILE                                                   PASS

File: Libraries/CommonFunctions.psm1 (hunk 1 above)
Comment (asofron) [reply Libraries/CommonFunctions.psm1]: [hyperv] [LISAv2 Test Results Summary]
[hyperv] Test Run On           : 12/17/2018 10:35:55
[hyperv] VHD Under Test        : Ubuntu_18.04.1.vhdx
[hyperv] Total Test Cases      : 1 (1 Pass, 0 Fail, 0 Abort)
[hyperv] Total Time (dd:hh:mm) : 0:0:15
[hyperv] XML File              : TestConfiguration
[hyperv] 
[hyperv]    ID TestCaseName                                                 TestResult TestDuration(in minutes) 
[hyperv] ------------------------------------------------------------------------------------------------------
[hyperv]     1 BVT-CORE-RELOAD-MODULES-SMP                                        PASS                15.66

File: Testscripts/Windows/DM-HOTADD-STRESS-APP.ps1 (hunk 3 above)
Comment (asofron) [reply Testscripts/Windows/DM-HOTADD-STRESS-APP.ps1]: [hyperv] [LISAv2 Test Results Summary]
[hyperv] Test Run On           : 12/17/2018 11:04:08
[hyperv] VHD Under Test        : Ubuntu_18.04.1.vhdx
[hyperv] Total Test Cases      : 1 (1 Pass, 0 Fail, 0 Abort)
[hyperv] Total Time (dd:hh:mm) : 0:0:10
[hyperv] XML File              : TestConfiguration
[hyperv] 
[hyperv]    ID TestCaseName                                                 TestResult TestDuration(in minutes) 
[hyperv] ------------------------------------------------------------------------------------------------------
[hyperv]     1 DYNAMIC-MEMORY-HOTADD-STRESS-APP                                   PASS                10.24 
[hyperv]

File: Libraries/CommonFunctions.psm1 [hunk 9]
```diff
@@ -2722,6 +2722,7 @@ function Convert-StringToUInt64{
 
 function Check-Result{
 	param(
+		[String] $User,
```
Comment (LiliDeng) [inline Libraries/CommonFunctions.psm1]: This method can be removed, I found there is the same method in BVT-CORE-RELOAD-MODULES.ps1 and only invoked in BVT-CORE-RELOAD-MODULES.ps1, so this method is unused?

**Comment by asofron** [comment]
@LiliDeng 

PS File | TestCases | Platform | PASS/FAIL
-- | -- | -- | --
Libraries/CommonFunctions.psm1 | BVT-CORE-RELOAD-MODULES-SMP | Azure / HyperV | PASS
  | STRESSTEST-RELOAD-MODULES-UP | Azure / HyperV | PASS
Testscripts/Windows/BVT-VERIFY-KVP-DAEMON-STATUS.ps1 | BVT-VERIFY-KVP-DAEMON-STATUS | Azure | PASS
Testscripts/Windows/DM-HOTADD-STRESS-APP.ps1 | DYNAMIC-MEMORY-HOTADD-STRESS-APP | Hyperv | PASS
Testscripts/Windows/FCOPY-DISABLE-ENABLE.ps1 | FCOPY-DISABLE-ENABLE | Hyperv | PASS
Testscripts/Windows/FCOPY-LARGE-FILE.ps1 | FCOPY-LARGE-FILE | Hyperv | PASS
Testscripts/Windows/FCOPY-NEGATIVE.ps1 | FCOPY-NEGATIVE | Hyperv | PASS
Testscripts/Windows/FCOPY-REPEATED-DELETE.ps1 | FCOPY-REPEATED-DELETE | Hyperv | PASS
Testscripts/Windows/Heartbeat-PausedCritical.ps1 | PAUSED-CRITICAL-HEARTBEAT | Hyperv | PASS
Testscripts/Windows/KDUMP-TestScript.ps1 | KDUMP-CRASH-SINGLE-CORE | Azure / HyperV | PASS
  | KDUMP-CRASH-SMP | Azure / HyperV | PASS
  | KDUMP-CRASH-NMI | Azure / HyperV | PASS
  | KDUMP-CRASH-AUTO-SIZE | Azure / HyperV | PASS
  | KDUMP-CRASH-16-CORES | Azure / HyperV | PASS
  | KDUMP-CRASH-DIFFERENT-VCPU | Azure / HyperV | PASS
Testscripts/Windows/KVP-MaxRead.ps1 | KVP-MAX-READ | Hyperv | FAIL
Testscripts/Windows/KVP-PushKeyValues.ps1 | KVP-PUSH-KEY-VALUES | Hyperv | PASS
Testscripts/Windows/Max-vCPU.ps1 | MAX-VCPU | Hyperv | PASS
Testscripts/Windows/SQM-Basic.ps1 | SQM-BASIC | Hyperv | PASS
Testscripts/Windows/STORAGE-HotUnplug.ps1 | STORAGE-VHDX-HOT-PLUG-UNPLUG-SCSI | Hyperv | PASS
Testscripts/Windows/STRESS-MTU-Netvsc-Reload.ps1 | STRESSTEST-CHANGE-MTU-RELOAD-NETVSC | Hyperv | PASS
Testscripts/Windows/Timesync-VDSO.ps1 | TIMESYNC-VDSO | Hyperv | FAIL


2 Failure are not related to root removal : 
Timesync-VDSO - VDSO not supported by kernel
KVP-MaxRead - Fails get correct version if not RHEL distro

File: Libraries/CommonFunctions.psm1 (hunk 9 above)
Comment (asofron) [reply Libraries/CommonFunctions.psm1]: will remove

**Pull request merged**